Jenkinsfile

```diff
@@ -1962,6 +1962,28 @@ pipeline {
       }
     }
 
+    stage('L2: Megatron Change Partitions') {
+      when {
+        anyOf {
+          branch 'main'
+          changeRequest target: 'main'
+        }
+      }
+      failFast true
+      steps{
+        sh "python examples/nlp/language_modeling/megatron_change_num_partitions.py \
+            --model_file \
+            /home/TestData/nlp/megatron_gpt/125M/megatron_gpt.nemo \
+            --target_file \
+            /home/TestData/nlp/megatron_gpt/125M/test-split.nemo \
+            --tensor_model_parallel_size \
+            1 \
+            --target_tensor_model_parallel_size \
+            2"
```

Comment:
Small nitpick, but the case we're most interested in is from larger num partitions -> smaller num partitions. I'd rather we test that case in our CI.

Reply:
Changed CI to 2->1